src/components/VideoPlayer/index.tsx

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
+import uniqueIDForVideoWithoutReport from '@components/VideoPlayerContexts/PlaybackContext/uniqueID';
 import BaseVideoPlayer from './BaseVideoPlayer';
 import type {VideoPlayerProps} from './types';
 
 function VideoPlayer(props: VideoPlayerProps) {
+    const {noReportID} = uniqueIDForVideoWithoutReport();
```

Comment:
I think we should use another name like mockReportID or fakeReportID

```suggestion
    const {noReportID} = uniqueIDForVideoWithoutReport();
```